### PR TITLE
Install make for ci

### PR DESCRIPTION
**Pull request by kurtismullins**
### Description of Changes

Fixes an issue where `make` is missing during certain CI tests and causes CI to fail.

#### Changes:

* Use the CI image for all tests
* Ran `black` and corrected formatting on one file to unblock CI in the lint stage.

#### Issue: 

- https://issues.redhat.com/browse/PROJQUAY-68

**TESTING**

- CI should no longer fail on executing `make`

**BREAKING CHANGE**

n/a

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: Dockerfile.cirun [hunk 1]
```diff
@@ -1,9 +1,14 @@
 FROM quay-ci-base
+
+ENV ENCRYPTED_ROBOT_TOKEN_MIGRATION_PHASE remove-old-fields
```
Comment (thomasmckay) [inline Dockerfile.cirun]: This should no longer be needed

Comment (kurtismullins) [reply Dockerfile.cirun]: @thomasmckay It's not yet removed from `master`. Should we wait until that change comes through (next week?) to make it?

Comment (thomasmckay) [reply Dockerfile.cirun]: sorry, thought it was in master already. nm this comment
